### PR TITLE
fix(3.2): ReactorDubbo3TripleStub.mustache add schema registry

### PR DESCRIPTION
**Pull request by caoyanan666**
## What is the purpose of the change

The ReactorDubbo3TripleStub.mustache template does not add schema registration. In this case, the gateway cannot pull all interface information and cannot make generalized calls. It should be consistent with Dubbo3TripleStub.mustache and add schema registration.
image

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Check if is necessary to patch to Dubbo 3 if you are work on Dubbo 2.7
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Add some description to [dubbo-website](https://github.com/apache/dubbo-website) project if you are requesting to add a feature.
- [ ] GitHub Actions works fine on your own branch.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by caoyanan666** [comment]
@AlbumenJ  
image
Is the dubbo-samples-triple-reactor module not in the project? How should I fix this unit test?

**Comment by AlbumenJ** [comment]
> @AlbumenJ image Is the dubbo-samples-triple-reactor module not in the project? How should I fix this unit test?

https://github.com/apache/dubbo-samples/tree/master/3-extensions/protocol/dubbo-samples-triple-reactor

File: dubbo-compiler/src/main/resources/ReactorDubbo3TripleStub.mustache [hunk 1]
```diff
@@ -54,6 +54,7 @@ public final class {{className}} {
     private static final StubServiceDescriptor serviceDescriptor = new StubServiceDescriptor(SERVICE_NAME,{{interfaceClassName}}.class);
 
     static {
+        org.apache.dubbo.rpc.protocol.tri.service.SchemaDescriptorRegistry.addSchemaDescriptor(SERVICE_NAME,{{outerClassName}}.getDescriptor());
```
Comment (chickenlj) [inline dubbo-compiler/src/main/resources/ReactorDubbo3TripleStub.mustache:57]: Does this mean that `outClassName` must appear in the `.proto` definition file?

Comment (chickenlj) [reply dubbo-compiler/src/main/resources/ReactorDubbo3TripleStub.mustache:57]: @caoyanan666 What if user does not set `outClassName` in his `.proto` file?

Comment (caoyanan666) [reply dubbo-compiler/src/main/resources/ReactorDubbo3TripleStub.mustache:57]: Yes, must definition.  https://github.com/apache/dubbo/blob/3.2/dubbo-compiler/src/main/resources/Dubbo3TripleStub.mustache also requires definition outClassName 
image

**Comment by caoyanan666** [comment]
@AlbumenJ PTAL

**Review by AlbumenJ — APPROVED** [review]

**Comment by AlbumenJ** [comment]
@EarthChen PTAL

**Review by EarthChen — APPROVED** [review]
LGTM

**Pull request merged**